netsim/devices/srlinux.yml

```diff
@@ -32,6 +32,7 @@ features:
       unnumbered: True
     ipv6:
       lla: True
+  stub_loopback: True
```

Comment:
This feature flag becomes ```stub_loopback: {}```, the 'True' value is lost in translation

Reply:
I think you made a wrong conclusion. If features.stub_loopback would have been set, referencing it wouldn't have reset it to `{}`.

Reply:
What I mean is: If you print out the 'features' dict at the place where it checks for stub_loopback, it shows as '{}' regardless whether I put ```stub_loopback: True``` or ```stub_loopback: False``` in the device yaml. Same is true for 'bfd' for example

There seems to be a difference in the levels - because the 'lla' flag does get retained as 'True' for example

This is confusing to me - in your example you have ```devices.iosv.features.stub_loopback: False```, so somehow that does get retained?

Reply:
That looks like a bug. Will check what's going on, and change the logic to consider device flag before the system default, so you'll be able to set the device flag to **True**.